### PR TITLE
chore: separate header logic from screens

### PR DESCRIPTION
**Pull request by FrantisekMichalSebestyen**
Seperated Header from Views 
Needed for page transitions to work eventually.

### CODE REVIEW

**Review by igor-kamil — CHANGES_REQUESTED** [review]
see comments. overall 👌

File: resources/js/app.js [hunk 1]
```diff
@@ -9,13 +9,42 @@ import App from './App.vue'
...
+      title: 'Artwork detail'
+    }
+  },
+    {
+      path: '/edit/:id',
+      component: Detail,
+      meta: {
+        'edit' : true,
+        'code': HEADER_CODES.NONE,
+        title: 'Artwork detail'
```
[the remark this answers is not in the document]
Comment (FrantisekMichalSebestyen) [reply resources/js/app.js]: resolved with prettier :)

File: resources/js/app.js [hunk 2]
```diff
@@ -9,13 +9,42 @@ import App from './App.vue'
...
+  {
+    path: '/collection', component: Collection, meta: {
+      code: HEADER_CODES.NONE,
+      title: 'SNG Codes'
+    }
+  },
+  {
+    path: '/:id?', component: Collection, meta: {
+      code: HEADER_CODES.NONE,
+      title: 'SNG Codes'
```
[the remark this answers is not in the document]
Comment (FrantisekMichalSebestyen) [reply resources/js/app.js]: ✅

**Review by igor-kamil — APPROVED** [review]
👌 done

**Pull request merged**